src/hotspot/share/classfile/classLoaderDataGraph.cpp

```diff
@@ -310,7 +310,7 @@ void ClassLoaderDataGraph::modules_do_keepalive(void f(ModuleEntry*)) {
   ClassLoaderDataGraphIterator iter;
   while (ClassLoaderData* cld = iter.get_next()) {
     // Keep the holder alive.
-    (void)cld->holder();
+    cld->keep_alive();
```

Comment:
Does comment need changing or removing?

Reply:
I agree. There's no need to talk about the holder here.